sources/US/WY/walk-in-2018.json

```diff
@@ -0,0 +1,18 @@
+{
+   "name" : "WY WIH Areas - 2018",
+   "url": "https://opendata.arcgis.com/datasets/5d87c370e473423c9ba7ae30a9598a16_0.zip",
+   "filenameInZip": "Walk_In_Hunting_Boundaries.shp",
+   "species": [
+      "all"
+   ],
+   "attribution": "Wyoming Fish and Wildlife",
+   "properties": {
+      "id": "OBJECTID",
+      "name": "LABEL"
```

Comment:
property LABEL not found in source feature error processing feature: {'geometry': {'type': 'Polygon', 'coordinates': (((-108.601201, 44.980289), (-108.598616, 44.980244), (-108.598626, 44.981564), (-108.601133, 44.981597), (-108.601201, 44.980289)),)}, 'type': 'Feature', 'id': '691', 'properties': OrderedDict([(u'OBJECTID', 713), (u'WIA_NO', 14), (u'COUNTY', u'Big Horn'), (u'Type', u'CLOSED'), (u'Label', u'CLOSED'), (u'AccessDate', u'CLOSED'), (u'Waterfowl', u''), (u'Pheasant', u''), (u'Sharp_tail', u''), (u'SageGrouse', u''), (u'BlueGrouse', u''), (u'SandhillCr', u''), (u'Doves', u''), (u'Partridge', u''), (u'Turkey', u''), (u'Rabbit', u''), (u'Antelope', u''), (u'Deer', u''), (u'Elk', u''), (u'Moose', u''), (u'BighornShe', u''), (u'MountainLi', u''), (u'BlackBear', u''), (u'Footnote_L', u''), (u'Footnote_1', u''), (u'Footnote_2', u''), (u'Footnote_3', u''), (u'Footnote_4', u''), (u'Footnote_5', u''), (u'Footnote_6', u''), (u'Footnote_7', u''), (u'Footnote_8', u''), (u'Footnote_9', u''), (u'Footnote10', u''), (u'Footnote_S', u''), (u'Acres', 0), (u'Species', u'CLOSED'), (u'Shape__Are', 3.347411335e-06), (u'Shape__Len', 0.007723241201864)])}